gcloud/storage/test_bucket.py

```diff
@@ -758,6 +758,7 @@ def test_get_logging_w_prefix(self):
         resp_to_reload = before
         connection = _Connection(resp_to_reload)
         bucket = self._makeOne(NAME, connection)
+        bucket._reload_properties()
```

Comment:
Given that we aren't relying on implicit GET / PATCH any longer, shouldn't the tests just drop the whole connection dance, and setup / make assertions only about local state?